MumbleSharp/Audio/Codecs/Opus/NativeMethods.cs

```diff
@@ -34,15 +34,25 @@ namespace MumbleSharp.Audio.Codecs.Opus
     /// <summary>
     /// Wraps the Opus API.
     /// </summary>
-    internal class NativeMethods
+    public class NativeMethods
     {
+
         static NativeMethods()
+        {
+            LoadLibrary();
```

Comment:
Maybe remove this completely?

Reply:
I left it in because I did not want to break the way the library (and its examples) work, with the thought in mind that it could be upstreamed eventually. However, if we just plan to use our fork forever, it could be removed.

Reply:
We probably are going to make even more changes anyway + I already kinda changed that part already - https://github.com/martindevans/MumbleSharp/blob/master/MumbleSharp/Audio/Codecs/Opus/NativeMethods.cs#L41
++ Doing any logic in static constructors is a very bad practice - its triggered by things like Assembly.GetTypes (and is very annoying when trying to do unit tests)

Reply:
I agree on it being bad practice. I considered completely rewriting it, which is probably what we should do - but did not get around to yet.